src/lib.rs

```diff
@@ -72,12 +72,15 @@
 #[macro_use]
 extern crate error_chain;
 extern crate futures;
+extern crate http;
 #[macro_use]
```

Comment:
There are no macros imported from hyper anymore; this can be removed.

Reply:
yes, i missed that